src/plugins/intel_cpu/tests/functional/subgraph_tests/src/inplace_edge.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils/cpu_test_utils.hpp"
+#include "shared_test_classes/base/layer_test_utils.hpp"
+#include "ov_models/utils/ov_helpers.hpp"
+#include "ov_models/builders.hpp"
+
+using namespace CPUTestUtils;
+using namespace InferenceEngine;
+
+namespace SubgraphTestsDefinitions {
+// If a node (CumSum) with constant parents has several non-constant nodes (Eltwises) than the edge is broken.
+// The fix is to check node type - is should be Input.
+// Subgraph:
+/*
+ *            Constant  Constant
+ *                 \    /
+ *                  \  /
+ *                 CumSum
+ *  Parameter      /   \     Parameter
+ *        \       /     \       /
+ *         \     /       \     /
+ *         Eltwise       Eltwise
+ *               \       /  
+ *                Eltwise
+ *                   |
+ *                 Result
+ */
+
+class NonInputInPlaceTest : public testing::WithParamInterface<InferenceEngine::Precision>,
+                            virtual public LayerTestsUtils::LayerTestsCommon {
```

Comment:
`LayerTestsUtils::LayerTestsCommon` is the legacy API base class, which is deprecated, please use `SubgraphBaseTest` instead.

Reply:
addressed